src/Plugin_Command.php

```diff
@@ -152,6 +152,7 @@ public function status( $args ) {
 	 *     **icons**: Plugin's Icon Image Link
 	 *     **active_installs**: Plugin's Number of Active Installs
 	 *     **contributors**: Plugin's List of Contributors
+	 *     **url**: Plugin's URL on wordpress.org 
```

Comment:
CS: There's a trailing space here.

Reply:
Good catch. Fixed in the rebased 26862c0.